rel_ext.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, src_filename_or_examples):
     @staticmethod
     def read_examples(src_filename):
         examples = []
-        with gzip.open(src_filename) as f:
+        with gzip.open(src_filename, encoding='utf8') as f:
```

Comment:
We can do this, but we have to specify `mode='rt'` as well. In that case, we can change 
```
fields = line[:-1].decode('utf-8').split('\t')
```
to 
```
fields = line[:-1].split('\t')
```
This loses Python 2 compatibility, but perhaps it is time to give up on that.